engine/src/main/java/org/hibernate/validator/internal/xml/BootstrapConfigurationImpl.java

```diff
@@ -164,6 +173,7 @@ public String toString() {
 		sb.append( ", messageInterpolatorClassName='" ).append( messageInterpolatorClassName ).append( '\'' );
```

Comment:
Not really. We have a String not a Type or a Class. We could implement one to work on String but I don't think it's really worth it in this case.